Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -186,7 +186,7 @@ extension KeyboardShortcuts {
 
 					NSAlert.showModal(
 						for: self.window,
-						message: "This keyboard shortcut cannot be used as it's already used by the “\(menuItem.title)” menu item."
+						message: String(format: "This keyboard shortcut cannot be used as it's already used by the “%@” menu item.".localized, menuItem.title)
```

Comment:
It looks like the most prop solution is using `localizedStringWithFormat`.

https://stackoverflow.com/a/47389291/64949
https://www.raywenderlich.com/250-internationalizing-your-ios-app-getting-started#toc-anchor-003

Reply:
Done